.gitignore

```diff
@@ -3,4 +3,13 @@ sources/*
 !sources/README.md
```

Comment:
gitignore just specifies what files will or will not be added to your git repository when you do `git add -A`. For example, in MediaWiki if you do `git add -A` in the MediaWiki install directory, it won't add your `LocalSettings.php` file to the repository because the `.gitignore` file says to ignore that file. Now, you could force your repository to add that particular file by doing `git add LocalSettings.php`, but that's another story.

In meza there are some instances where all files in a directory are ignored (like `sources/*`), but a specific file is explicitly un-ignored (`!sources/README.md`). Basically the reason for this README is to be a placeholder for that directory so git tracks its existence even when nothing else is in it. When meza is first pulled, there is nothing in `sources` (other than the README), but the directory is a useful place to download stuff to during install.